From: David Steele <steele@debian.org>
Date: Sun, 25 Jun 2017 19:30:38 -0400
Subject: Patch: Reproducible version message

---
 main.go | 13 ++-----------
 1 file changed, 2 insertions(+), 11 deletions(-)

diff --git a/main.go b/main.go
index 13514c1..434ec1c 100644
--- a/main.go
+++ b/main.go
@@ -95,21 +95,12 @@ func changePassword(args *argContainer) {
 // printVersion prints a version string like this:
 // gocryptfs v0.12-36-ge021b9d-dirty; go-fuse a4c968c; 2016-07-03 go1.6.2
 func printVersion() {
-	humanTime := "0000-00-00"
-	if i, _ := strconv.ParseInt(BuildTime, 10, 64); i > 0 {
-		t := time.Unix(i, 0).UTC()
-		humanTime = fmt.Sprintf("%d-%02d-%02d", t.Year(), t.Month(), t.Day())
-	}
 	buildFlags := ""
 	if stupidgcm.BuiltWithoutOpenssl {
 		buildFlags = " without_openssl"
 	}
-	built := fmt.Sprintf("%s %s", humanTime, runtime.Version())
-	if raceDetector {
-		built += " -race"
-	}
-	fmt.Printf("%s %s%s; go-fuse %s; %s\n",
-		tlog.ProgramName, GitVersion, buildFlags, GitVersionFuse, built)
+	fmt.Printf("%s %s%s; go-fuse %s\n",
+		tlog.ProgramName, GitVersion, buildFlags, GitVersionFuse)
 }
 
 func main() {
