From: David Steele <steele@debian.org>
Date: Sun, 25 Jun 2017 19:43:29 -0400
Subject: Fix new go syncmap path

---
 internal/fusefrontend_reverse/rfile.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/internal/fusefrontend_reverse/rfile.go b/internal/fusefrontend_reverse/rfile.go
index 742d697..ed60597 100644
--- a/internal/fusefrontend_reverse/rfile.go
+++ b/internal/fusefrontend_reverse/rfile.go
@@ -7,7 +7,7 @@ import (
 	"syscall"
 
 	// In newer Go versions, this has moved to just "sync/syncmap".
-	"golang.org/x/sync/syncmap"
+	"sync/syncmap"
 
 	"github.com/hanwen/go-fuse/fuse"
 	"github.com/hanwen/go-fuse/fuse/nodefs"
